testRequests.py

```diff
@@ -107,8 +107,13 @@ def getTestScript(PrepID, nEvents):
         inputfile = open(filename, 'r')
         outputfile = open(tmpfilename, 'w')
         for line in inputfile:
-            outputfile.write(re.sub('(.*RAWSIM.*-n) \d*( .*)', r'\1 %s\2'
-                                    % (nEvents), line))
+            outline = re.sub('(.*--eventcontent LHE.*-n) \d*( .*)', r'\1 %s\2'
+                                    % (nEvents), line)
+            outline = re.sub('(.*--eventcontent DQM.*-n) \d*( .*)', r'\1 %s\2'
```

Comment:
I used the script to on chained request prepid, so the test job end up having 6 cmsDriver commands, I am modifying only those that are not 'harvesting' (i.e. 1,2 and 4,5 in the list below):
(If I understand the question ? please correct me if not!)

```bash
cmsDriver.py Configuration/GenProduction/python/HIG-RunIIWinter15wmLHE-00601-fragment.py --fileout file:HIG-RunIIWinter15wmLHE-00601.root --mc --eventcontent LHE --datatier LHE --conditions MCRUN2_71_V1::All --step LHE --python_filename HIG-RunIIWinter15wmLHE-00601_1_cfg.py --no_exec --customise Configuration/DataProcessing/Utils.addMonitoring -n 100 || exit $? ;
```
```bash
echo "nothing" ;cmsDriver.py Configuration/GenProduction/python/HIG-RunIIWinter15wmLHE-00601-fragment.py --fileout file:HIG-RunIIWinter15wmLHE-00601.root --mc --eventcontent DQM --datatier DQM --conditions MCRUN2_71_V1::All --step LHE,USER:GeneratorInterface/LHEInterface/wlhe2HepMCConverter_cff.generator,GEN,VALIDATION:genvalid_all  --fileout file:HIG-RunIIWinter15wmLHE-00601_genvalid.root --mc -n 100 --python_filename HIG-RunIIWinter15wmLHE-00601_genvalid.py --dump_python --no_exec || exit $? ;
```
```bash
cmsDriver.py step2 --filein file:HIG-RunIIWinter15wmLHE-00601_genvalid.root --conditions MCRUN2_71_V1::All --mc -s HARVESTING:genHarvesting --harvesting AtJobEnd --python_filename HIG-RunIIWinter15wmLHE-00601_genvalid_harvesting.py --no_exec || exit $? ;
```
```bash
cmsDriver.py Configuration/GenProduction/python/ThirteenTeV/Hadronizer_TuneCUETP8M1_13TeV_generic_LHE_pythia8_cff.py --filein file:HIG-RunIIWinter15wmLHE-00601.root --fileout file:HIG-RunIISummer15GS-00577.root --mc --eventcontent RAWSIM --customise SLHCUpgradeSimulations/Configuration/postLS1Customs.customisePostLS1,Configuration/DataProcessing/Utils.addMonitoring --datatier GEN-SIM --conditions MCRUN2_71_V1::All --beamspot Realistic50ns13TeVCollision --step GEN,SIM --magField 38T_PostLS1 --python_filename HIG-RunIISummer15GS-00577_1_cfg.py --no_exec -n 100 || exit $? ;
```
```bash
echo "nothing" ;cmsDriver.py Configuration/GenProduction/python/ThirteenTeV/Hadronizer_TuneCUETP8M1_13TeV_generic_LHE_pythia8_cff.py --filein file:HIG-RunIIWinter15wmLHE-00601.root --fileout file:HIG-RunIISummer15GS-00577.root --mc --eventcontent DQM --datatier DQM --conditions MCRUN2_71_V1::All --beamspot Realistic50ns13TeVCollision --step GEN,VALIDATION:genvalid_all --magField 38T_PostLS1  --fileout file:HIG-RunIISummer15GS-00577_genvalid.root --mc -n 100 --python_filename HIG-RunIISummer15GS-00577_genvalid.py  --no_exec || exit $? ;
```
```bash
cmsDriver.py step2 --filein file:HIG-RunIISummer15GS-00577_genvalid.root --conditions MCRUN2_71_V1::All --mc -s HARVESTING:genHarvesting --harvesting AtJobEnd --python_filename HIG-RunIISummer15GS-00577_genvalid_harvesting.py --no_exec || exit $? ;
```